samples/wifi/thread_coex/README.rst

```diff
@@ -94,9 +95,9 @@ To enable different test modes, set up the following configuration parameters in
 .. note::
    You can also use ``menuconfig`` to configure ``Wi-Fi credentials``.
 
-   See :ref:`zephyr:menuconfig` in the Zephyr documentation for instructions on how to run ``menuconfig``.
+See :ref:`zephyr:menuconfig` in the Zephyr documentation for instructions on how to run ``menuconfig``.
```

Comment:
This looks like an unneeded change, to me. Menuconfig is only mentioned in the note, so additional information about it can stay inside the note as well. But not something that absolutely needs to be reverted, I don't think.